apple/Sources/FerrostarSwiftUI/Extensions/UniFFIExtensions.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  UniFFIExtensions.swift
+//  FerrostarCore
+//
+//  Created by Patrick Wolowicz on 27.09.24.
+//
+
+import FerrostarCoreFFI
+import FerrostarCore
+import CoreLocation
+import MapLibre
+
+extension FerrostarCore {
+    
+    public var isNavigating: Bool {
+        return self.state?.isNavigating ?? false
+    }
+}
+
+extension NavigationState {
+    public var isNavigating: Bool {
```

Comment:
I chose a very simple definition of isNavigating - I'm not sure about this though. Is the .idle tripState navigation for example?

Reply:
Yeah, this is correct.